src/components/landing_page/LandingPage.scss

```diff
@@ -29,8 +28,8 @@
 
 .Particles {
   width: 100%;
-  height: 100%;
-  position: fixed;
+  height: calc(100% - var(--footer-height));

```

Comment:
The way the footer was done on the landing page previously (transparent background) was a specific request by @luisebenkert. It looks like you are re-adding the dark border at the bottom of the landing page here, by deleting the <LandingPageFooter /> and using the Unauthorized Footer here instead.

Reply:
Yeah, but the footer is always hidden and you can only see it when scrolling down.

Reply:
That sounds good then 👍